docs/MAINTAINERS-guidelines.md

```diff
@@ -0,0 +1,51 @@
+# Introduction
+All Hyperledger projects MUST have a `MAINTAINERS.md` file at the top-level directory of the source code. This document will provide guidelines on what should be included in the `MAINTAINERS.md` file.
```

Comment:
Some repositories currently have a MAINTAINERS.rst file instead. I think we should allow that variation.

Reply:
Agree

Reply:
Changes made to reflect.